### PR TITLE
Update language samples to dump all data returned from server.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
According to an earlier discussion, we still want to show users a list of all
the features that will be returned by the server.  So, I added a sample
invocation to the help documentation that serves that purpose.

### CODE REVIEW

File: language/api/src/AllCommand.php [hunk 1]
```diff
@@ -41,6 +41,149 @@ protected function configure()
...
+      number: SINGULAR
+      person: PERSON_UNKNOWN
+      proper: PROPER
+      reciprocity: RECIPROCITY_UNKNOWN
+      tense: TENSE_UNKNOWN
+      voice: VOICE_UNKNOWN
+    dependencyEdge:
+      headTokenIndex: 1
+      label: NSUBJ
+    lemma: John
```
Comment (bshaffer) [inline language/api/src/AllCommand.php]: I do not think it's useful to have this much in the example output. How about a single token block with an ellipsis,  to indicate there will be more.

File: language/api/src/SyntaxCommand.php [hunk 2]
```diff
@@ -40,7 +40,122 @@ protected function configure()
...
+      person: PERSON_UNKNOWN
+      proper: PROPER
+      reciprocity: RECIPROCITY_UNKNOWN
+      tense: TENSE_UNKNOWN
+      voice: VOICE_UNKNOWN
+    dependencyEdge:
+      headTokenIndex: 1
+      label: NSUBJ
+    lemma: John
+  -
```
Comment (bshaffer) [inline language/api/src/SyntaxCommand.php]: Same here. We don't want the user to have to scroll up to get the full help text.

File: language/api/src/AllCommand.php (hunk 1 above)
Comment (SurferJeffAtGoogle) [inline language/api/src/AllCommand.php]: Done.

File: language/api/src/SyntaxCommand.php (hunk 2 above)
Comment (SurferJeffAtGoogle) [inline language/api/src/SyntaxCommand.php]: Done.

**Comment by SurferJeffAtGoogle** [comment]
Ready for another look.

**Pull request merged**